kpconv_torch/utils/trainer.py

```diff
@@ -1,5 +1,4 @@
-from os import makedirs, remove
-from os.path import exists, join
+import os
```

Comment:
I'm totally OK with these change. Are you *really* sure every occurrence of these functions has been modified?

Hint: use `git grep` on the file, for instance `git grep -a exists kpconv_torch/utils/trainer.py`.